doc/source/workflow/overview.md

```diff
@@ -0,0 +1,208 @@
+- `observe` current state of the cluster
+- `diff` against desired state
+- if necessary `act` to apply desired state
+
+![](../images/operator.svg)
+
+
+## Service Orchestrator
+
+`Service Orchestrator` is responsible for managing intra-graph traffic.
+It reads the inference graph structure from the `CRD` and when inference request is received it makes sure that it is passed to each node of the graph in the right order.
+
+It is because of the presence of `Service Orchestrator` that complex graph components like `routers`, `combiners` and output/input `transformers` are available in the `Seldon` world.
+
+![](../images/orchestrator.svg)
+
+`Service Orchestrator` is also responsible for providing many advance features out of the box:
+- `Jaeger` tracing
+- `Prometheus` metrics
+- request payload logging
+
+to just name a few.
+
+Read more about `Service Orchestrator` on its dedicated documentation [page](../graph/svcorch.html).
+
+
+## Metadata Provenance
+
+In `Seldon` we understand the importance of the Model Metadata.
+You can easily version your model and describe its expected inputs and outputs.
+
+These allow you to make connection to the platform you trained your model with (DVC, Pachyderm, ...)
+and know what inputs / outputs you can expect from your inference graph.
+
+![](../images/metadata.svg)
+
+Read more about metadata provenance on its dedicated documentation [page](../reference/apis/metadata.html).
+
+
+## Metrics with Prometheus
+
+Metrics is important aspect of serving ML inference models in production.
+Out of the box Seldon Core deployments expose standard metrics to [Prometheus](https://prometheus.io/) on the `Service Orchestrator`.
+
+![](../images/metrics.svg)
+
+Users can also define their custom metrics that will be exposed by models directly.
+To add metrics to Python models one can simply define `metrics` method:
+```python
+class Model:
+    ...
+
+    def metrics(self):
+        return [
+            # a counter which will increase by the given value
+            {"type": "COUNTER", "key": "mycounter", "value": 1},
+
+            # a gauge which will be set to given value
+            {"type": "GAUGE", "key": "mygauge", "value": 100},
+
+            # a timer which will add sum and count metrics - assumed millisecs
+            {"type": "TIMER", "key": "mytimer", "value": 20.2},
+        ]
+```
+
+Read more about metrics on its dedicated documentation [page](../analytics/analytics.html).
+
+
+## Distributed Tracing with Jaeger
+
+You can use Open Tracing to trace your API calls to Seldon Core. By default we support [Jaeger](https://www.jaegertracing.io/) for Distributed Tracing, which will allow you to obtain insights on latency and performance across each microservice-hop in your Seldon deployment.
+
+![](../images/tracing.svg)
+
+Read more about tracing on its dedicated documentation [page](../graph/distributed-tracing.html).
+
+
+## Why Seldon Core?
```

Comment:
May be worth adding the "Why not just wrapping your model with Flask? question here before, maybe instead of adding the slide as the slide is just a list. It may still be worth adding below the list you have (ie "other features of seldon core:"